Selenium.psm1

```diff
@@ -8,54 +8,118 @@ elseif($IsMacOS){
     $AssembliesPath = "$PSScriptRoot/assemblies/macos"
 }
 
+# Grant Execution permission to assemblies on Linux and MacOS 
+if($IsLinux -or $IsMacOS){
+    # Check if powershell is NOT running as root
+    $AssemblieFiles = Get-ChildItem -Path $AssembliesPath |Where-Object{$_.Name -eq 'chromedriver' -or $_.Name -eq 'geckodriver'}
+    foreach($AssemblieFile in $AssemblieFiles){
+        if($IsLinux){
+            $FileMod = stat -c "%a" $AssemblieFile.fullname
+        }
+        elseif($IsMacOS){
+            $FileMod = /usr/bin/stat -f "%A" $AssemblieFile.fullname
+        }
+
+        if($FileMod[2] -ne '5' -and $FileMod[2] -ne '7' ){
+            Write-Host "Granting $($AssemblieFile.fullname) Execution Permissions ..."
```

Comment:
The reason I used write host here is because I want to make sure the users know thst we are changing permissions on their system not only when they specify -Verbose so I think we should keep l the write-host.